cloud_run_jobs_execute_jobs_on_schedule/main.tf

```diff
@@ -70,7 +71,7 @@ resource "google_cloud_scheduler_job" "job" {
 
   http_target {
     http_method = "POST"
-    uri         = "https://${google_cloud_run_v2_job.default.location}-run.googleapis.com/apis/run.googleapis.com/v1/namespaces/${data.google_project.project.number}/jobs/${google_cloud_run_v2_job.default.name}:run"
+    uri         = "https://${google_cloud_run_v2_job.default.location}-run.googleapis.com/apis/run.googleapis.com/v1/namespaces/your-project-id/jobs/${google_cloud_run_v2_job.default.name}:run"
```

Comment:
Sorry for the misunderstanding. Please change this back to `${data.google_project.project.number}`.

My point is don't use the `data` block unless you need to. In this case you need it.

Reply:
10-4